test/e2e/common.go

```diff
@@ -62,6 +62,22 @@ func withPVCBinding(mountPath string, pvcName string) podOption {
 	}
 }
 
+func withContainerPort(port int) podOption {
+	return func(p *corev1.Pod) {
+		p.Spec.Containers[0].Ports = []corev1.ContainerPort{
+			{
+				ContainerPort: int32(port),
+			},
+		}
+	}
+}
+
```

Comment:
It doesn't look like we are using `withContainerPort` here, so I don't think we need to add this function?